packages/components/Callout/macos/CalloutWindow.swift

```diff
@@ -1,21 +1,14 @@
 import Foundation
 import AppKit
+import Carbon.HIToolbox
```

Comment:
I need this one header for "kVK_Escape" on line 45. I could just.. hardcode the key code (53) to avoid pulling in a carbon header, but I'm not sure if that's better.

Reply:
no hardcode :)

Reply:
Carbon will never die!